desktop/lib.go

```diff
@@ -98,6 +103,13 @@ func start() {
 		}
 	}()
 
+	go func() {
+		err = fetchUserData()
+		if err != nil {
+			log.Errorf("error while fetching user data: %v", err)
+		}
+	}()
```

Comment:
I would maybe just simplify this to..

```go
go fetchUserData()
```

since we are already logging the error in `fetchUserData`

Reply:
Done.